keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -292,14 +329,94 @@ Checkpointing containers will be possible again.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Currently no.
+Currently the test will automatically be skipped if the feature is not enabled.
+
+### Rollout, Upgrade and Rollback Planning
+
+Does not apply as the feature is an additional API endpoint with no
+dependencies on other functionality. If it is not enabled via the feature
+gate it will return `404 page not found`. If it is not enabled in the
+underlying container engine a `500` will be returned with an error
+message from the container engine. If it is enabled the API endpoint exists
+if disabled then it does not exist. No planning necessary.
+
+Documented at <https://kubernetes.io/docs/reference/node/kubelet-checkpoint-api/>
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+At this point it is still a kubelet only API endpoint and has no dependencies
```

Comment:
it depends on the CRI

Reply:
We can tweak to say that it is a kubelet API endpoint that needs the corresponding configured CRI to support the checkpoint API.

Reply:
Will do.

Reply:
That is fine, and it should be laid out in the risks and mitigation section, as well as the appropriate dependencies section in the PRR questionnaire. Here, the whole point of this question is to think through the worst possible scenario. See the [template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md) (just make sure to look in the raw mode, since a lot of those suggestions are in comment sections.